tests/test_caching.py

```diff
@@ -0,0 +1,52 @@
+#    Copyright 2015-2017 ARM Limited
```

Comment:
That sounds fine to my inexpert ears - actually it looks like the [tests](https://github.com/ARM-software/trappy/blob/master/tests/test_copyright.py#L44) require "ARM Limited" to be in the copyright line so you'll need to change that test if you just want to mark something as solely Google copyright (which would also be fine IIUC).